utility/src/main/java/org/oppia/util/parser/ImprovedBulletSpan.kt

```diff
@@ -0,0 +1,75 @@
+package org.oppia.util.parser
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.Path
+import android.graphics.Path.Direction
+import android.text.Layout
+import android.text.Spanned
+import android.text.style.LeadingMarginSpan
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.util.R
+
+/**
+ * Copy of [android.text.style.BulletSpan] from android SDK 28 with removed internal code.
```

Comment:
Please expand on what this provides that's different from the Android version. 'Improved' isn't very clear, and I'm really hesitant to pull code out of the SDK since it will quickly become out of date with Android's version of it.

Reply:
Changed file name to `CustomBulletSpan` also, added one more comment line.

`This class helps us to customise bullet radius, gap width and offset present in rich-text.`